app/policies/schools/target_policy.rb

```diff
@@ -1,9 +1,11 @@
 module Schools
   class TargetPolicy < ApplicationPolicy
-    def create?
+    def update?
```

Comment:
This should be `content?`
`target controller` does not have an `update` methord.